views/index.pug

```diff
@@ -0,0 +1,20 @@
+html
+    head
+        body
+            h1 Kim's Convenience
+                
+            h2  Developed By
+                li Ins Choi 
+                li Kevin White
+            h3 Starring
```

Comment:
This area was not looking for you to list this yourself it was looking for you to generate the information.  For example:
 body
    h1= showdata.title

    h2 Developed By
    ul
      for developer in showdata.developedBy
        li= developer

    h2 Starring
    ul
      for star in showdata.starring
        li= star